.github/workflows/lint-pr.yml

```diff
@@ -261,3 +273,52 @@ jobs:
+          history_cli_feat=$(git log --pretty=format:"%s %h" $head...HEAD -- ./packages/cli | grep "^feat" || true)
+          echo "$history_cli_feat"
+          history_toolkit_feat=$(git log --pretty=format:"%s %h" $head...HEAD -- ./packages/vscode-extension | grep "^feat" || true)
+          history_core_feat=$(git log --pretty=format:"%s %h" $head...HEAD -- ./packages/fx-core | grep "^feat" || true)
+          history_sdk_feat=$(git log --pretty=format:"%s %h" $head...HEAD -- ./packages/sdk | grep "^feat" || true)
+          history_sdk_react_feat=$(git log --pretty=format:"%s %h" $head...HEAD -- ./packages/sdk-react | grep "^feat" || true)
+          history_dotnet_sdk_feat=$(git log --pretty=format:"%s %h" $head...HEAD -- ./packages/dotnet-sdk | grep "^feat" || true)
+          echo -e '\nFx-core feat commits:\n' "$history_core_feat" '\n\nCLI feat commits:\n' "$history_cli_feat" '\n\nExtension-toolkit feat commits:\n' "$history_toolkit_feat" '\n\nSDK feat commits:\n' "$history_sdk_feat" '\n\nSDK React feat commits:\n' "$history_sdk_react_feat" '\n\n.Net SDK feat commits:\n' "$history_dotnet_sdk_feat" '\n'>> "$tempfile"
+          history_cli_fix=$(git log --pretty=format:"%s %h" $head...HEAD -- ./packages/cli | grep "^fix" || true)
+          history_toolkit_fix=$(git log --pretty=format:"%s %h" $head...HEAD -- ./packages/vscode-extension | grep "^fix" || true)
+          history_core_fix=$(git log --pretty=format:"%s %h" $head...HEAD -- ./packages/fx-core | grep "^fix" || true)
+          history_sdk_fix=$(git log --pretty=format:"%s %h" $head...HEAD -- ./packages/sdk | grep "^fix" || true)
+          history_sdk_react_feat=$(git log --pretty=format:"%s %h" $head...HEAD -- ./packages/sdk-react | grep "^fix" || true)
+          history_dotnet_sdk_feat=$(git log --pretty=format:"%s %h" $head...HEAD -- ./packages/dotnet-sdk | grep "^fix" || true)
+          echo -e '\nFx-core fix commits:\n' "$history_core_fix" '\n\nCLI fix commits:\n' "$history_cli_fix" '\n\nExtension-toolkit fix commits:\n' "$history_toolkit_fix" '\n\nSDK fix commits:\n' "$history_sdk_fix"  '\n\nSDK React fix commits:\n' "$history_sdk_react_fix" '\n\n.Net SDK fix commits:\n' "$history_dotnet_sdk_fix" '\n'>> "$tempfile"
+          cat "$tempfile"
+          cat ./packages/vscode-extension/PRERELEASE.md >> "$tempfile"
+          mv "$tempfile" ./packages/vscode-extension/PRERELEASE.md
+          git checkout -b changelog/${{ github.event.inputs.source-branch }}
+      
+      - name: setup user info
+        working-directory: ./
+        run: |
+          user_info=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/users/${{ github.actor }})
+          git config --global user.email $(echo $user_info | jq -r '.name')
+          git config --global user.name $(echo $user_info | jq -r '.name')
+          git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/OfficeDev/teams-toolkit.git
```

Comment:
there may have permission error in github_token
may need to add 
permission 
    content: write